tests/test_lm/test_integration/test_jsrun/test_5/cmds

```diff
@@ -1,3 +1,3 @@
-# Task needs to be assigned nodes 1 and 2, physical core 1 of each node, HW thread 1 of each physical core
+# Task needs to be assigned node 1, physical core 8, HW thread 1, 0 GPUs
```

Comment:
Same as above (marked *)
Indices start from 0 or 1? Just to make sure we are consistent.

Reply:
The node list starts from 1, CPUs and GPUs from 0